subsys/bluetooth/shell/iso.c

```diff
@@ -247,9 +253,11 @@ static int cmd_cig_create(const struct shell *sh, size_t argc, char *argv[])
 			return -ENOEXEC;
 		}
 
-		param.latency = latency;
+		param.c_to_p_latency = latency;
+		param.p_to_c_latency = latency;
 	} else {
-		param.latency = 10;
+		param.c_to_p_latency = 10;
+		param.p_to_c_latency = 10;
```

Comment:
Suggest to avoid duplicating literal values

Reply:
agreed